blueprints/data-solutions/vertex-mlops/README.md

```diff
@@ -0,0 +1,76 @@
+# MLOps with Vertex AI - Infra setup
+
+## Introduction
+This example implements the infrastructure required to deploy an end-to-end [MLOps process](https://services.google.com/fh/files/misc/practitioners_guide_to_mlops_whitepaper.pdf) using [Vertex AI](https://cloud.google.com/vertex-ai) platform. 
+
+##  GCP resources
+The blueprint will deploy all the required resources to have a fully functional MLOPs environment containing:
+
+- GCP Project  to host all the resources
+- Isolated VPC network and a subnet to be used by Vertex and Dataflow (using a Shared VPC is also possible). 
+- Firewall rule to allow the internal subnet communication required by Dataflow
+- Cloud NAT required to reach the internet from the different computing resources (Vertex and Dataflow)
+- GCS buckets to host Vertex AI and Cloud Build Artifacts. By default the buckets will be regional and should match the Vertex AI region for the different resources (i.e. Vertex Managed Dataset) and processes (i.e. Vertex trainining)
+- BigQuery Dataset where the training data will be stored. This is optional, since the training data could be already hosted in an existing BigQuery dataset.
+- Service account (`mlops-[env]@`) with the minimum permissions required by Vertex and Dataflow
+- Service account (`github@`) to be used by Workload Identity Federation, to federate Github identity (Optional). 
+- Secret to store the Github SSH key to get access the CICD code repo.
+
+![MLOps project description](./images/mlops_projects.png "MLOps project description")
+
+## Pre-requirements
+
+### User groups
+
+Assign roles relying on User groups is a way to decouple the final set of permissions from the stage where entities and resources are created, and their IAM bindings defined. These groups should be created before launching Terraform.
+
+We use the following groups to control access to resources:
+
+- *Data Scientits* (gcp-ml-ds@<company.org>). They create ML pipelines in the experimentation environment.
+- *ML Engineers* (gcp-ml-eng@<company.org>). They manage and run the different environments, with access to all resources in order to troubleshoot possible issues with pipelines. 
+
+These groups are not suitable for production grade environments. You can configure the group names through the `groups`variable. 
+
+##  Instructions
+###  Deploy the experimentation environment
+
+- Create a `terraform.tfvars` file and specify the variables to match your desired configuration. You can use the provided `terraform.tfvars.sample`  as reference.
+- Make sure you have the right authentication setup (application default credentials, or a service account key)
+- Run `terraform init` and `terraform apply`
+- It is possible that some errors like `googleapi: Error 400: Service account xxxx does not exist.` appears. This is due to some dependencies with the Project IAM authoritative bindings of the service accounts. In this case, re-run again the process with `terraform apply`
```

Comment:
I am under the impression that all dependencies can be properly fixed without the need to run tf apply more than once. Can you list the with is the SA facing dependencies issues? 

Happy to take this offline on a VC.

Reply:
You are right, now this work OK. I've removed this line

Reply:
Fixed